app/components/Showcases/Barter.jsx

```diff
@@ -0,0 +1,724 @@
+                        fee_asset_types.splice(fee_asset_types.indexOf(key), 1);
+                    }
+                }
+            }
+
+            return {asset_types, fee_asset_types};
+        };
+
+        let from = getAssetTypes(from_account, from_error);
+        let to = getAssetTypes(to_account, to_error);
+
+        return {
+            from_asset_types: from.asset_types || [],
+            to_asset_types: to.asset_types || [],
+            from_fee_asset_types: from.asset_fee_types || [],
+            to_fee_asset_types: to.fee_asset_types || []
+        };
+    }
+
+    addFromAmount() {
+        this.state.from_barter.push("");
+        this.setState({from_barter: this.state.from_barter});
+    }
+
+    addToAmount() {
+        this.state.to_barter.push("");
+        this.setState({to_barter: this.state.to_barter});
+    }
+
+    onSubmit(e) {
+        e.preventDefault();
+        this.setState({from_error: null, to_error: null});
+        let sendAmount;
+        let transfer_list = [];
+
+        this.state.from_barter.forEach(item => {
+            const asset = item.from_asset;
+            let amount = item.from_amount;
+            sendAmount = new Asset({
+                real: amount,
+                asset_id: asset.get("id"),
+                precision: asset.get("precision")
+            });
+            transfer_list.push({
+                from_account: this.state.from_account.get("id"),
+                to_account: this.state.to_account.get("id"),
+                amount: sendAmount.getAmount(),
+                asset_id: asset.get("id"),
+                memo: this.state.memo
+                    ? new Buffer(this.state.memo, "utf-8")
+                    : this.state.memo,
+                propose: this.state.from_account, //  propose
+                feeAsset: item.from_feeAsset
+                    ? item.from_feeAsset.get("id")
+                    : "1.3.0"
+            });
+        });
+
+        this.state.to_barter.forEach(item => {
+            const asset = item.to_asset;
+            let amount = item.to_amount;
+            sendAmount = new Asset({
+                real: amount,
+                asset_id: asset.get("id"),
+                precision: asset.get("precision")
+            });
+            transfer_list.push({
+                from_account: this.state.to_account.get("id"),
+                to_account: this.state.from_account.get("id"),
+                amount: sendAmount.getAmount(),
+                asset_id: asset.get("id"),
+                memo: this.state.memo
+                    ? new Buffer(this.state.memo, "utf-8")
+                    : this.state.memo,
+                propose: this.state.to_account, //  propose
+                feeAsset: item.to_feeAsset
+                    ? item.to_feeAsset.get("id")
+                    : "1.3.0"
+            });
+        });
+
+        AccountActions.transfer_list(transfer_list)
```

Comment:
Replace this call with 

`console.log(transfer_list)` 

for testing

Reply:
I added console.log(transfer_list)